backend/infrahub/core/diff/model/path.py

```diff
@@ -163,6 +163,10 @@ class EnrichedDiffAttribute(BaseSummary):
     def __hash__(self) -> int:
         return hash(self.name)
 
+    @property
+    def num_properties(self) -> int:
+        return len(self.properties)
```

Comment:
We seem to be using "count" for this in other places instead of "num". It could be mostly related to counting stuff coming from queries though. I don't know if we use "num" in many other places as well though, so it's perhaps not really consistent. Don't know what's right or wrong here, but something to consider.